src/app/pages/courses/pipes/orderby.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Course } from 'app/shared/models/course.model';
+
+@Pipe({
+    name: 'orderByDate'
+})
+
+export class OrderByPipe implements PipeTransform {
+    public transform(courses: Course[], direction: string) {
```

Comment:
This pipe is intended to be universal and domain-agnostic. Try omitting business-related variable names such as `course` in universal components.